frontend/public/components/masthead-toolbar.jsx

```diff
@@ -260,19 +295,21 @@ class MastheadToolbar_ extends React.Component {
             {flags[FLAGS.OPENSHIFT] && <ToolbarItem>
               <Dropdown
                 isPlain
+                key="applauncher"
                 onSelect={this._onApplicationLauncherDropdownSelect}
                 toggle={
                   <DropdownToggle aria-label="Application Launcher" iconComponent={null} onToggle={this._onApplicationLauncherDropdownToggle}>
                     <ThIcon />
                   </DropdownToggle>
                 }
                 isOpen={isApplicationLauncherDropdownOpen}
-                dropdownItems={this._renderMenuItems(this._launchActions())}
+                dropdownItems={this._renderMenuItems(this._launchActions(), serviceMeshItem)}
               />
             </ToolbarItem>}
             <ToolbarItem>
               <Dropdown
                 isPlain
+                key="help"
```

Comment:
Why `key`?

Reply:
> Why `key`?

I was getting a 'need unique key' warnings until I added these.

Reply:
Removed it here, only needed it in `<DropdownItem ../>`'s